pyproject.toml

```diff
@@ -250,6 +248,9 @@ user = [
 [tool.poetry.scripts]
 pymedphys = "pymedphys.__main__:main"
 
+[tool.poetry.group.dev.dependencies]
+ipykernel = "^6.27.1"
```

Comment:
Can we make sure this gets included as a dev requirement the same way others are:
https://github.com/pymedphys/pymedphys/pull/1818/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L124

Reply:
Actually I don't know how this was added to the dependency list :sweat_smile:.

I removed it.

Reply:
:)